tests/utils/shippable/sanity.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -o pipefail -eux
+
+declare -a args
+IFS='/:' read -ra args <<< "$1"
+
+group="${args[1]}"
+
+if [ "${BASE_BRANCH:-}" ]; then
+    base_branch="origin/${BASE_BRANCH}"
+else
+    base_branch=""
+fi
+
+if [ "${group}" == "extra" ]; then
+    ../internal_test_tools/tools/run.py --color
+    exit
+fi
+
+case "${group}" in
```

Comment:
You don't need this anymore since you reduced to one group. (Right now, all other sanity tests are not run!)

You can use this file instead, which is optimized for one group: https://github.com/ansible-collections/community.crypto/blob/main/tests/utils/shippable/sanity.sh

Reply:
yep, already done:) thanks